### PR TITLE
lmp-boot-firmware: cleanup and minor changes

### PR DESCRIPTION
**Pull request by quaresmajose**

### CODE REVIEW

**Review by angolini — APPROVED** [review]

**Review by MrCry0 — APPROVED** [review]

File: meta-lmp-base/recipes-bsp/lmp-boot-firmware/lmp-boot-firmware.bb [hunk 1]
```diff
@@ -54,16 +54,15 @@ do_install() {
             for file in `ls ${D}${nonarch_base_libdir}/${FIRMWARE_DEPLOY_DIR}/`; do
                 version="${version}-`md5sum ${D}${nonarch_base_libdir}/${FIRMWARE_DEPLOY_DIR}/${file} | cut -d' ' -f1`"
             done
-
-	    # limit the length of version
-	    version="`echo ${version} | md5sum | cut -d' ' -f1`"
+            # limit the length of version
+            version="`echo ${version} | md5sum | cut -d' ' -f1`"
```
Comment (ricardosalveti) [inline meta-lmp-base/recipes-bsp/lmp-boot-firmware/lmp-boot-firmware.bb:59]: nit, can you drop these changes or isolate them in another patch (tab x spaces)?

**Comment by ricardosalveti** [comment]
Can you please extend the commit message on every change to avoid having changes with subject only descriptions?

**Review by igoropaniuk — APPROVED** [review]
LGTM

File: meta-lmp-base/recipes-bsp/lmp-boot-firmware/lmp-boot-firmware.bb (hunk 1 above)
Comment (quaresmajose) [reply meta-lmp-base/recipes-bsp/lmp-boot-firmware/lmp-boot-firmware.bb:60]: done

**Comment by quaresmajose** [comment]
> Can you please extend the commit message on every change to avoid having changes with subject only descriptions?

Added descriptions in all except the indentation fix

**Review by MrCry0 — APPROVED** [review]
LGTM

**Review by ricardosalveti — APPROVED** [review]
LGTM

**Pull request merged**